build/src/utils/image-content-extractor.js

```diff
@@ -533,8 +533,19 @@ async function getImageInfo(imageTagOrContainerName) {
     if(isContainerName(imageTagOrContainerName)) {
         image = await asyncUtils.spawn('docker', ['inspect', "--format='{{.Image}}'", imageTagOrContainerName.trim()], { shell: true, stdio: 'pipe' });
     }
-    const imageNameAndDigest = await asyncUtils.spawn('docker', ['inspect', "--format='{{index .RepoDigests 0}}'", image], { shell: true, stdio: 'pipe' });
-    const [name, digest] = imageNameAndDigest.trim().split('@');
+    // If image not yet published, there will be no repo digests, so set to N/A if that is the case
+    let name, digest;
+    try {
+        const imageNameAndDigest = await asyncUtils.spawn('docker', ['inspect', "--format='{{index .RepoDigests 0}}'", image], { shell: true, stdio: 'pipe' });
+        [name, digest] = imageNameAndDigest.trim().split('@');
+    } catch(err) {
+        if(err.result.indexOf('Template parsing error') > 0) {
+            name = 'N/A';
+            digest = 'N/A';
+        } else {
+            throw err;
+        }
+    }
```

Comment:
FYI - This is a bug I ran into while testing history markdown generation locally. Works file in CI. To see what ends up in history, run:

```bash
build/vscdc info --registry mcr.microsoft.com --registry-path vscode/devcontainers --release main jekyll
```

The result will be in containers/jekyll/history/dev.md